docs/prover/Report/CallResolutionTable.md

```diff
@@ -0,0 +1,56 @@
+CallResolutionTable
+=================
+
+The `CallResolutionTable` shows information about all the summarized calls in the program.
+It helps the user to better understand decisions made by the Prover- which calls in the rule code got inlined, which were replaced by summaries, and why.
+
+Attributes
+------
+
+ * **Caller**- The caller of the call. Always resolved by the Prover.
+
+ * **Callee**- The callee of the call. For internal calls, always resolved. For external calls, might be unresolved, depending on linking, summarization, and more.
+See below.
+
+ * **Call Site**- source information of the call itself. Shows the location of the call (which file, which line), and a snippet from the source code (the invocation itself).
+
+ * **Summary**- what summary got applied for the call.
+
+ * **Comments**- a list of comments about the resolution of the callee (which as mentioned above, might be unresolved), the applied summarization.
+See below.
+
+
+The Callee
+------
+
+The callee is composed of two elements which should be taken into account: 1. Callee contract 2. Callee sighash.
+The callee contract is the target contract of the call, the callee sighash is the sighash of the invoked function.
+As mentioned above, for internal calls, the callee will always be resolved by the tool, while for external calls, it’s not always the case.
+Here all the cases for unresolved callees are introduced:
+
+ * Fully resolved callee: In such a case, the summary will be applied iff the application policy of the summary is `ALL` 
+(see {doc}`The Methods Block — Certora Prover Documentation 0.0 documentation <summary-types>`).
```

Comment:
I am not sure how to link to another page inside the documentation, but I would like to link to: https://docs.certora.com/en/latest/docs/cvl/methods.html?highlight=summary#summary-types

Reply:
You usually want to use the ref syntax to link to a specific section:
```
(any-label-you-want)=
## Section header

...

You can link to it using {ref}`any-label-you-want` (which will display "Section Header" in the text) or {ref}`display text <any-label-you-want>`.
```

There is also the "doc" syntax to link to an entire page:
```
Link to {doc}`/docs/prover/cli/options` (note the missing .md)
```

Reply:
I have tried to use your suggestion, and pushed the code. I see it doesn't work.
I did not get the desired linking. What am I missing?

Reply:
> You usually want to use the ref syntax to link to a specific section:
> 
> ```
> (any-label-you-want)=
> ## Section header
> 
> ...
> 
> You can link to it using {ref}`any-label-you-want` (which will display "Section Header" in the text) or {ref}`display text <any-label-you-want>`.
> ```
> 
> There is also the "doc" syntax to link to an entire page:
> 
> ```
> Link to {doc}`/docs/prover/cli/options` (note the missing .md)
> ```

I have tried to use your suggestion, and pushed the code. I see it doesn't work.
I did not get the desired linking. What am I missing?

Reply:
It looks like it's a different problem now:
```
docs/prover/Report/CallResolutionTable.md: WARNING: document isn't included in any toctree
```

This indicates that the file hasn't been linked anywhere; you need to add it to the `index.md` file.

You can see the error messages by clicking through the "details" link under the failing check, although you have to log in to readthedocs.com first.